.github/workflows/preview-docs.yml

```diff
@@ -39,37 +39,22 @@ jobs:
 
   teardown:
     if: |
-      github.event.action == 'closed' &&
+      github.event.pull_request.merged == true &&
```

Comment:
This is GitHub's recommendation in https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-pull_request-workflow-when-a-pull-request-merges

Reply:
Do you not still want to teardown when the PR is closed but not merged?

Reply:
Oh, good point. We want to teardown whenever the PR is closed. But for some reason it hasn't been working. My hypothesis is us switching to use the merge queue is relevant, although I haven't confirmed that.